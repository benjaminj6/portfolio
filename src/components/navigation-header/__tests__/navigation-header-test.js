/* eslint-env jest */
import { h } from 'preact'
import { shallow } from 'preact-render-spy'
import NavigationHeader from '..'

// TODO -- up test coveragge for this component
test('Header initial snapshot', () => {
  const tree = shallow(<NavigationHeader />)
  expect(tree).toMatchSnapshot()
})
