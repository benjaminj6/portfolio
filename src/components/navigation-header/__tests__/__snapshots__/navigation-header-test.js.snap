// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<div
  data-nav-open={false}
  class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723 NavigationHeader"
>
  <header class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723">
    <button
      onClick={[Function anonymous]}
      class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723"
    >
      open!
    </button>
  </header>
  <nav class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723">
    <ul class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723">
      <li class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723">
        <a
          href="#projects"
          onClick={[Function anonymous]}
          class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723"
        >
          Projects
        </a>
      </li>
      <li class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723">
        <a
          href="#blog"
          onClick={[Function anonymous]}
          class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723"
        >
          Latest From the Blog
        </a>
      </li>
      <li class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723">
        <a
          href="#about"
          onClick={[Function anonymous]}
          class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723"
        >
          About Me
        </a>
      </li>
      <li class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723">
        <a
          href="#contact"
          onClick={[Function anonymous]}
          class="jsx-2085888330 jsx-986440723 jsx-2085888330 jsx-986440723"
        >
          Contact Me
        </a>
      </li>
    </ul>
  </nav>
  <JSXStyle
    styleId="4031427020"
    css=".__jsx-style-dynamic-selector{--transition:350ms ease-in-out;}header.__jsx-style-dynamic-selector{position:fixed;height:var(--header-height);display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:end;-webkit-justify-content:flex-end;-ms-flex-pack:end;justify-content:flex-end;top:0;left:0;right:0;padding:1rem;background:transparent;z-index:999;}button.__jsx-style-dynamic-selector:hover{cursor:pointer;}nav.__jsx-style-dynamic-selector{position:fixed;top:0;bottom:0;left:0;right:0;background:var(--white);border-bottom:1px solid var(--gray-normal);-webkit-transform:translateY(calc(-100% + var(--header-height)));-ms-transform:translateY(calc(-100% + var(--header-height)));transform:translateY(calc(-100% + var(--header-height)));-webkit-transition:background var(--transition), -webkit-transform var(--transition);-webkit-transition:background var(--transition), transform var(--transition);transition:background var(--transition), transform var(--transition);}[data-nav-open].__jsx-style-dynamic-selector nav.__jsx-style-dynamic-selector{-webkit-transform:translateY(0);-ms-transform:translateY(0);transform:translateY(0);-webkit-transition:background var(--transition), -webkit-transform var(--transition);-webkit-transition:background var(--transition), transform var(--transition);transition:background var(--transition), transform var(--transition);background:var(--accent-primary);}ul.__jsx-style-dynamic-selector{margin-top:var(--header-height);padding:1rem;}li.__jsx-style-dynamic-selector{font-size:1.5rem;padding:2.5rem;width:100%;text-align:center;}a.__jsx-style-dynamic-selector{color:var(--white);text-decoration:none;font-weight:bold;}body{overflow:initial;}"
    dynamic={
      Array [
        "initial"
      ]
    }
  >
  </JSXStyle>
  <JSXStyle
    styleId="2085888330"
    css=""
  >
  </JSXStyle>
</div>

`;
